api/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>appframework</artifactId>
-		<version>2.5-SNAPSHOT</version>
+		<version>2.6-SNAPSHOT</version>
```

Comment:
@darius, wanted a major new release, because the changes removed support for 1.x line